### ISSUE TITLE
miniMover console make error

### ISSUE THREAD

**Issue by drjackk** [reality-boy/miniMover#10]
In miniMoverConsole, make fails with the error:
"make: *** No rule to make target '../miniMoverLib/xyzprinterlist.cpp', needed by 'xyzprinterlist.o'.  Stop."

The makefile calls for 'xyzprinterlist'.
The filenames are actually 'XYZPrinterList.cpp' & 'XYZPrinterList.h'
By modifying the makefile with the correct filenames, make runs without error.

**Comment by exaiyo** [comment]
I encountered this as well.  Are you compiling on a UNIX system which is case sensitive?  Does miniMover even support linux? I'm surprised that this information is not in the main page readme

**Comment by drjackk** [comment]
Yes, miniMoverConsole has had Linux support since version 0.9.5.
It is unfortunate that Linux support is not mentioned in any of the docs.
I think I discovered miniMover supported Linux in a forum somewhere.  Otherwise, who would know.

I got the code to compile and to give me the usage information, but priorities changed and I've never actually tried it on my da Vinci Mini printer.

**Comment by joselvazquez** [comment]
Just modify the xyzprinterlist.* to XYZPrinterList.* in the makefile. Then it works. It might be that the author system didn't mind about case sensitivity
